docs/cardanoNuFiConnect.md

```diff
@@ -225,5 +225,14 @@ For users with NuFi extension installed, there are no specific actions required.
 Simply access `window.cardano.nufi` from anywhere as it is not controlled by
 the NuFi Widget SDK.
 
+## Whitelist
+
+### NuFiConnect mainnet
```

Comment:
I'd consider clarifying that `localhost` is whitelisted by default for the sake of development

Reply:
Are you sure about CSP? I have been using an extension just to bypass CORS, I'm not aware of any extension to bypass CSP